build-parent/pom.xml

```diff
@@ -195,6 +198,23 @@
                 </exclusions>
             </dependency>
 
+            <!-- JNA dependency is fixed due to transitive dependency issue on testcontainers-->
+            <dependency>
+                <groupId>net.java.dev.jna</groupId>
+                <artifactId>jna</artifactId>
+                <version>${jna.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers</artifactId>
+                <version>${test-containers.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>postgresql</artifactId>
+                <version>${test-containers.version}</version>
+            </dependency>
```

Comment:
I see you removed one of the artifact, we don't use it?

Reply:
Yes, the junit related artifact wasn't being used anywhere